src/sweeper/__main__.py

```diff
@@ -4,60 +4,105 @@
 sweeper
 
 Usage:
-  sweeper sweep duplicates <table_name> --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path>]
-  sweeper sweep empties <table_name> --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path>]
-  sweeper sweep invalids <table_name> --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path>]
+  sweeper sweep duplicates --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]
+  sweeper sweep empties --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]
+  sweeper sweep invalids --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]
+  sweeper sweep all-checks --workspace=<workspace> [--verbose --try-fix --save-report=<report_path> --backup-to=<backup_path> --table-name=<table_name>]  
 
```

Comment:
I like the new usage style. I wonder if all-checks should be dropped and sweeper sweep does everything?

Reply:
yeah, that could work.  i'll give it a shot.

Reply:
@steveoh how would I check for that in the args?  Here's the current pattern: 
`    if args['duplicates']:
        call_sweeper(args, "DuplicateTest")
    elif args['invalids']:
        call_sweeper(args, "InvalidsTest")
    elif args['empties']:
        call_sweeper(args, "EmptyTest")
    elif args['all-checks']:`

Reply:
i think by testing that all of the others are empty? Yea that edit works.

Reply:
i think by testing that all of the others are empty? Yea that edit works.